exercises/C-comparison-operators/exercise.js

```diff
@@ -16,6 +16,24 @@ let personA = "Daniel";
 let personB = "Irina";
 let sameName; // finish this statement
 
+if (studentCount > mentorCount){
+  moreStudentsThanMentors = true;
+} else {
+  moreStudentsThanMentors = false
+}
+
+if (roomMaxCapacity > studentCount){
+  enoughSpaceInRoom = true;
+} else {
+  enoughSpaceInRoom = false
+}
+
+if (personA == personB) {
+  sameName = true;
+} else {
+  sameName = false;
+}
+
```

Comment:
This works well. You could do this more concisely by assigning the result of the comparison directly to the variable, i.e. 
let moreStudentsThanMentors = studentCount>mentorCount;

Reply:
Done